.pipelines/jobs/OneBranchNuGet.yml

```diff
@@ -56,15 +55,11 @@ jobs:
           artifactName: 'drop_build_arm64'
           targetPath: '$(Build.SourcesDirectory)/arm64'
             
-      - task: NuGetCommand@2
+      - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
```

Comment:
Did we change the type of task here, and why is the command changed from 'pack' to 'custom'?

Reply:
NuGetCommand@2 works for the custom nuget commands too - I've switched back. Rather than trying to get all the input parameters to play nicely with the task I just used a custom nuget command which I copied out of a working pipeline log.